### ISSUE TITLE
Feature request: show new plugins in a separate category

### ISSUE THREAD

**Issue by maximbaz** [KELiON/cerebro#218]
Besides "Installed" and "Available" plugin categories I'd love to see the "New" category, showing plugins that have been just recently created. As a user I would like to monitor new plugins that are appearing every now and then, but I don't want to scroll through the entire list every time.

It could either be based on the plugin's creation date (if this info can be queried), or we could locally cache plugin names that I've seen, and match that local list of names with the list of results loaded from `registry.npmjs.com`.

/cc @BrainMaestro

**Comment by BrainMaestro** [comment]
Yeah I like that idea. I think I prefer the local plugin cache method. The only issue with that is a plugin that has been around for over a month can appear as new for a user that does not check their plugins often. Is that acceptable?

**Comment by maximbaz** [comment]
I think that totally makes sense, when that user will finally check their plugins after 1 month, they are still interested in the question "what's new, what has changed since last month?".

**Comment by dar5hak** [comment]
Instead of a separate category, how about just bubbling up the new plugins to the top of the **Available** category and indicating them by a :star2: or something? That also gives way to a future possibility of :fire: plugins.
